cairo_zero/kakarot/precompiles/kakarot_precompiles.cairo

```diff
@@ -46,76 +48,81 @@ namespace KakarotPrecompiles {
     ) {
         alloc_locals;
 
-        // Input must be at least 4 + 3*32 bytes long.
-        let is_input_invalid = is_nn(99 - input_len);
+        // Input must be at least MIN_EVM_ENCODED_STARKNET_CALL_BYTES bytes long to be valid.
+        let is_call_invalid = is_nn(MIN_EVM_ENCODED_STARKNET_CALL_BYTES - (input_len + 1));
```

Comment:
maybe add a comment about the fact that because of gas cost, `input_len < 2**128`